elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -99,7 +72,17 @@ public EntityDictionary() {
      * @return binding class
      */
     public String getBinding(Class<?> entityClass) {
-        return bindEntityToJsonApi.get(lookupEntityClass(entityClass));
+        return entity(entityClass).jsonApi;
+    }
+
+    /**
+     * Get the Database Manager for an entity
+     *
+     * @param entityClass entity name
+     * @return List of relationship names for entity
+     */
+    public DatabaseManager getDatabaseManager(Class<?> entityClass) {
```

Comment:
The EntityManager will implement a DatabaseManager interface that wraps multiple subordinate DatabaseManager into a single transaction.  However, rolling back a partial commit will be difficult.
Assuming there are 2 databases:  DB1 and DB2.  A commit on the global transaction would do
* Commit DB1
* Commit DB2 - fails
* Rollback DB2
* Reverse commit on DB1

If the reverse commit on DB1 fails, we would be left in an inconsistent state.